### PR TITLE
Fixed issue not attribute tab_width error

### PR DESCRIPTION
**Pull request by jamrizzi**
Fixes issue #43

### CODE REVIEW

**Comment by erikrose** [comment]
Thanks for the report and the patch! Can you share with me a test case (even a manual one) that reproduces it?

**Comment by jamrizzi** [comment]
```sh
git clone https://github.com/jamrizzi/sphinx-hello-world.git
cd sphinx-hello-world
make
```

**Comment by erikrose** [comment]
Aha, it appears to be a new behavior that appeared sometime after Sphinx 1.6.5. If I build your project with that version, it builds fine. Continuing to chase...

**Comment by jamrizzi** [comment]
Awesome, thanks

**Comment by jamrizzi** [comment]
This issue has now bled into readthedocs builds. They must have updated the sphinx version about 1.6.5. It is urgent that this is fixed. Please accept my pull request. It fixes the bug.

File: sphinx_js/renderers.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ class JsRenderer(object):
 
     """
     def __init__(self, directive, app, arguments=None, content=None, options=None):
+        if not hasattr(directive.state.document.settings, 'tab_width'):
+            directive.state.document.settings.tab_width = 4
```
Comment (erikrose) [inline sphinx_js/renderers.py]: Still trying to get to the bottom of why this fails. Anything against a default tab width of 8? I see that's the default in the docutils code.

**Comment by erikrose** [comment]
I've got one more PR to review, and then I'll do a release. Thanks for contributing!

File: sphinx_js/renderers.py (hunk 1 above)
Comment (jamrizzi) [reply sphinx_js/renderers.py]: No prob, I don't really care what the default is. The default didn't noticeably effect anything.

Comment (jamrizzi) [reply sphinx_js/renderers.py]: No prob, I don't really care what the default is. The default didn't noticeably effect anything.

**Pull request merged**